argo/providers/vai/vai_eventing_server_unit_test.go

```diff
@@ -225,15 +224,7 @@ var _ = Context("VaiEventingServer", func() {
 						Outputs: map[string]*aiplatformpb.PipelineTaskDetail_ArtifactList{
 							"a-model": {
 								Artifacts: []*aiplatformpb.Artifact{
-									{
-										Uri:         "gs://some/where",
-										SchemaTitle: "tfx.PushedModel",
-										Metadata: &structpb.Struct{
-											Fields: map[string]*structpb.Value{
-												"pushed": structpb.NewNumberValue(1),
-											},
-										},
-									},
+									artifact("gs://some/where"),
```

Comment:
Why is this the only one using the `artifact` function?

Reply:
this has now been restructured